python_modules/dagster/dagster/_core/definitions/remote_asset_graph.py

```diff
@@ -48,23 +52,28 @@
     from dagster._core.selector.subset_selector import DependencyGraph
 
 
+@whitelist_for_serdes
+@record
+class SelectAssetNode:
+    selector: RepositorySelector
+    asset: Annotated[
+        "ExternalAssetNode", ImportFrom("dagster._core.remote_representation.external_data")
+    ]
```

Comment:
was having a hard time naming this thing - the named variant of (location, asset) tuple 
"scoped" was another idea

Reply:
I like `RepositoryScopedAssetNode` -- it's possible that we actually want this to be generic though? i.e. `RepositoryScopedEntityNode[T_EntityNode]`